### DISCUSSION TITLE
lets encrypt certification expired issue

### DISCUSSION THREAD

**Discussion by dekanayake** [aws/aws-lambda-rust-runtime#350] [General]
hi, I'm using rust lambda for one of my production systems. I have a client service that calls HTTPS REST API. It was working without any issues, but today suddenly it started to fail with the following error.

error sending request for url (https://xx.xx.com/api/f43b0181-6ee9-4539-b3e9-f7f37a95f962/delivery/v1/search?q=(number1:650): error trying to connect: error:14090086:SSL routines:ssl3_get_server_certificate:certificate verify failed:s3_clnt.c:1264: (certificate has expired) . I think this is failing due to the lets encrypt DST Root CA X3 root certificate expiery.

I tried to fix this issue by attaching the root certificate Signed by ISRG Root X1 to reqwest client builder no luck. Could you please help me to fix this issue? Thanks.

**Comment by johansmitsnl** [comment]
When you test with https://www.ssllabs.com/ssltest/ does it say all good with the certificate? Looks like your API does not have the ssl certificates (or intermediate) correctly installed or the time of your machine is wrong.
